imagebuildah/build.go

```diff
@@ -190,7 +198,7 @@ func warnOnUnsetBuildArgs(node *parser.Node, args map[string]string) {
 				argHasValue = argFound[argName]
 			}
 			if _, ok := args[argName]; !argHasValue && !ok {
-				logrus.Warnf("missing %q build argument. Try adding %q to the command line", argName, fmt.Sprintf("--build-arg %s=<VALUE>", argName))
+				logger.Warnf("missing %q build argument. Try adding %q to the command line\n", argName, fmt.Sprintf("--build-arg %s=<VALUE>", argName))
```

Comment:
The `\n` was for `fmt.Fprintf()`, which doesn't add one by default, but `logrus`'s formatters always include it, so we don't need that.